tests/runtest.proj

```diff
@@ -412,6 +412,21 @@ namespace $([System.String]::Copy($(Category)).Replace(".","_").Replace("\","").
 
   </Target>
 
+  <Target Name="RunPerfTests" Condition="'$(Performance)'=='true'">  
+    <Message Text="Executing steps for perf tests" Importance="High"/>  
+    
+    <!-- generate project.json for runtime dependency -->  
+    <MSBuild Projects="$(MSBuildProjectFile)"  
+             Targets="CreateTestRuntimeJsonFile"/>  
+     
+    <!-- Package the product -->  
+    <MSBuild Projects="$(MSBuildProjectFile)" Targets="CreateTestOverlay" />  
```

Comment:
This is to copy the product to the payload directory so that we can package it as a correlation payload